### ISSUE TITLE
Double work records

### ISSUE THREAD

**Issue by karljonaskarlsson** [BetaMasaheft/Documentation#2253]
It appears that the work record that I just created was created twice, as LIT6762IntroEgzNagsa (https://betamasaheft.eu/works/LIT6762IntroEgzNagsa/main#t1?hi=nagsa) and as LIT6763IntroEgzNagsa (https://betamasaheft.eu/works/LIT6763IntroEgzNagsa/main#t1?hi=nagsa). I suggest that the latter is kept, as this is the file that I could download and that have been edited (see https://github.com/BetaMasaheft/Works/pull/906).

**Comment by eu-genia** [comment]
Will delete the doublet, make sure you only click "create" once )

**Comment by eu-genia** [comment]
@thea-m while checking I see that there is also LIT6761PPUndoing and LIT6760PPUndoing - do you need both? There is also LIT6757PPUndoing...

**Comment by thea-m** [comment]
LIT6761PPUndoing and LIT6757PPUndoing refer to different prayers. LIT6760PPUndoing  was also an accidental doublet (I'm pretty sure that neither Jonas nor I clicked twice - sometimes this still happens...). Thank you for noticing!

**Comment by eu-genia** [comment]
Dear @MassimoVilla , thank you very much for creating new records, can you please confirm that you are working an all of these
![image](https://user-images.githubusercontent.com/17987728/198705693-ab21c84d-55be-4dba-a7e8-9d8f9af69071.png)
there seem to have been 11 records for MiracleGMQ on 21.10 and 2 more on 24.10 - just to check that there are 13 miracles and you are filling them all out.
Thank you!

**Comment by MarcinKrawczuk** [comment]
No, it's a mistake, the PRS13930 must be deleted, it is incorrect anyway
because of the _ sign...

pt., 28 paź 2022 o 20:25 Eugenia ***@***.***> napisał(a):

**Comment by MassimoVilla** [comment]
> Dear @MassimoVilla , thank you very much for creating new records, can you please confirm that you are working on all of these ![image](https://user-images.githubusercontent.com/17987728/198705693-ab21c84d-55be-4dba-a7e8-9d8f9af69071.png) there seem to have been 11 records for MiracleGMQ on 21.10 and 2 more on 24.10 - just to check that there are 13 miracles and you are filling them all out. Thank you!

Dear Eugenia, yes I confirm that I created all records. They are now complete. I just need one more approval and they will be merged
